infra/build/functions/build_lib.py

```diff
@@ -387,7 +387,8 @@ def get_docker_build_step(image_names,
                           directory,
                           use_buildkit_cache=False,
                           src_root='oss-fuzz',
-                          architecture='x86_64'):
+                          architecture='x86_64',
+                          cache_name=''):
```

Comment:
nit: replace `cache_name` with `cache_image` to be clearer.

Reply:
done